libs/dal/src/repositories/log/log.repository.ts

```diff
@@ -1,14 +1,8 @@
-import { BaseRepository, Omit } from '../base-repository';
-import { LogEntity } from './log.entity';
+import { BaseRepository } from '../base-repository';
+import { LogEntity, LogDBModel } from './log.entity';
 import { Log } from './log.schema';
-import { Document, FilterQuery } from 'mongoose';
 
-class PartialLogEntity extends Omit(LogEntity, ['_environmentId', '_organizationId']) {}
-
-type EnforceEnvironmentQuery = FilterQuery<PartialLogEntity & Document> &
```

Comment:
How the enforcement is happening right now? Maybe just missed it

Reply:
answered here: https://github.com/novuhq/novu/pull/2890#discussion_r1116646908